src/pages/_app.tsx

```diff
@@ -1,33 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { CacheProvider, EmotionCache } from '@emotion/react';
-import App, { AppProps } from 'next/app';
-import { ThemeProvider, CssBaseline } from '@mui/material';
-
-import createEmotionCache from '../utility/createEmotionCache';
-import theme from '../styles/theme/theme';
-import '../styles/globals.css';
-
-const clientSideEmotionCache = createEmotionCache();
-
-interface QJAppProps extends AppProps {
-  emotionCache: EmotionCache;
-}
-
-class QJApp extends App<QJAppProps> {
-  render() {
-    const emotionCache = clientSideEmotionCache;
-    const { Component, pageProps } = this.props;
-
-    return (
-      <CacheProvider value={emotionCache}>
-        <ThemeProvider theme={theme}>
-          <CssBaseline />
-          <Component {...pageProps} />
-        </ThemeProvider>
-      </CacheProvider>
-    );
-  }
-}
+import { AppProps } from 'next/app';
+
+/**
+ * QJApp function component.
```

Comment:
I'm a bit fuzzy on next/react at this point. What's the difference between this and the Home component in `index.tsx`?

Reply:
The files and the components within them serve different purposes. 

At a glance:
`_app.tsx` is for customizing the App component. (Layout styles, global styles, etc.)
`index.tsx` is the main file for the homepage of the application. (What we see when the app opens)
`_document.tsx` is for customizing the default HTML document structure for all pages. (meta tags, HTML head, etc)

The files don't look much different as we haven't made the full pages yet.